### DISCUSSION TITLE
Autocomplete shouldn't insert parenthesis if there are already some

### DISCUSSION THREAD

**Discussion by etiennebacher** [posit-dev/positron#4907] [Feedback]
When replacing a function call using autocomplete, an extra pair of parenthesis will be added, which is unnecessary since there already are some:

[Screencast from 2024-10-04 22-04-05.webm](https://github.com/user-attachments/assets/79f9ce1c-de19-4c97-9fc1-c7411c942229)

I suppose this is a bug, but I don't know the behavior in python for instance

**Comment by etiennebacher** [comment]
Sorry, duplicate of https://github.com/posit-dev/positron/issues/2574 and https://github.com/posit-dev/positron/issues/4441

**Reply by juliasilge** [reply]
No worries! Thanks for the report.
